package.json

```diff
@@ -27,6 +27,8 @@
     "build:typecheck": "tsc -b -v $(node scripts/get-composite-ts-projects.js)",
     "check:packages": "yarn install --immutable --immutable-cache",
     "clean": "rimraf \"{apps,packages}/*/{build*,*.tsbuildinfo}\" \"coverage\" \".serverless*\"",
+    "contentful:migrate:dryrun": "yarn ctf-migrate up --all --dry-run",
```

Comment:
I'd expose these inside the contentful package

Reply:
Done.